scripts/generate_crds_manifests.sh

```diff
@@ -92,8 +91,6 @@ metadata:
   labels:
     app: package-server-manager
   annotations:
-    include.release.openshift.io/self-managed-high-availability: "true"
```

Comment:
Why was this annotation removed as well?

Reply:
Because they're opposite profiles. The self managed profile (the default cvo profile) shouldn't apply to the manifests that are specifically just for the ibm manifests. Take a look at our other deployment manifests that specifically include them, this change just makes those match.

For the manifests where we just have one copy of the manifest, both annotations apply.